Sourcing/ArrayDataProvider.swift

```diff
@@ -71,31 +71,32 @@ open class ArrayDataProvider<Element>: ArrayDataProviding, DataModifying {
         self.sectionIndexTitles = sectionIndexTitles
         self.headerTitles = headerTitles
     }
+    
     /**
      Reconfigures the dataSource with new data.
      
      - parameter array: flat array.
      - parameter updates: diff of the new data.
-     - parameter triggerdByTableView: flag if the change of data is already set in the TableView.
-    */
-    public func reconfigure(with array: [Element], updates: [DataProviderUpdate<Element>]? = nil, triggerdByTableView: Bool = false) {
-        reconfigure(with: [array], updates: updates, triggerdByTableView: triggerdByTableView)
+     - parameter updateView: flag if the views displaying the data should update. When your view is alredy up to date pass `false`.
```

Comment:
- parameter updateView: flag if the view displaying the data should be updated. Pass `false` if your view is already up to date. Defaults to `true`.